crates/bevy_color/src/color_difference.rs

```diff
@@ -0,0 +1,13 @@
+//! Module for calculating distance between two colors in the same color space.
+
+/// Calculate the distance between this and another color as if they were coordinates
+/// in a Euclidean space. Alpha is not considered in the distance calculation.
+pub trait EuclideanDistance: Sized {
```

Comment:
Definitely feels like a `bevy_math` trait that ended up here 😅 I'm okay to keep this in here for now, but we may want to think about how to represent / expose this later.

Reply:
ok